packages/core/src/sass/app/_common/components/settings-language.scss

```diff
@@ -8,10 +8,10 @@
         display: grid;
         grid-template-columns: repeat(4, minmax(90px, 1fr));
         grid-template-rows: repeat(5, auto);
-        grid-gap: 16px;
+        grid-gap: 1.5rem;
```

Comment:
why not 1.6rem?

Reply:
This looks fine as well. Ill go with 1.6rem